tests/codeception/common/_support/FixtureHelper.php

```diff
@@ -58,6 +58,30 @@ public function globalFixtures()
     }
 
     /**
+     * Workaround used to load custom fixtures for the test
+     * @link https://github.com/Codeception/Codeception/issues/2258
+     * @link https://github.com/yiisoft/yii2/issues/5575
+     * @param array $fixtureConfig
+     */
+    public function loadFixturesForTest(array $fixtureConfig)
+    {
+        $fixtures = ArrayHelper::merge(
```

Comment:
Ok, but how do I can manipulate fixtures when they are already loaded?
If I want to merge some fixtures? E.g. to append something to the global one, but not to duplicate its code?

If there are no `fixtures()` and `globalFixtures()` methods, I have to deal with mixture of `Fixture` objects and arrays. It will produce an ugly and a redundant code to do so. Remember that `loadFixtures()` will drop any previous data in the table...

I think     
    $fixtures = ArrayHelper::merge(
        $I->globalFixtures(),
        $I->fixtures(),
        ['user' => [
           'class' => UserFixture::className(),
           'dataFile' => '@tests/codeception/frontend/functional/fixtures/signup_cest.php',
       ]]
    )
is much more elegant and clear way to manipulate fixtures.

If you worrying about number of methods we will expose to Codeception (methods that will be available through `$I->`) we can decrease this number by moving texture loading routine into `FixtureHelper` like I propose here (`FixtureHelper::loadFixturesForTest()`).